### ISSUE TITLE
Share failures

### ISSUE THREAD

**Issue by jduckles** [carpentries/community-cookbook#8]
From [Wirawan](https://carpentries.topicbox.com/groups/champions/Tcbfb4e502ec6f4a7-M5a0897b125afffb59807fc42)
> Similarly, stories of failures or challenges are rarely published. If we can carefully present them (without shaming the failing parties), they can provide useful lessons for other people.

**Comment by mchiapello** [comment]
I strongly agree. Failure stories represent a great way of learning and not repeat some common mistakes.
